docs/Catalog/Packaging.md

```diff
@@ -0,0 +1,226 @@
+     file = "${params.outdir}/ro-crate-metadata.json"
+     overwrite = true
+     agent {
+       name = "John Doe"
+       orcid = "https://orcid.org/0000-0000-0000-0000"
+     }
+      license = "https://spdx.org/licenses/MIT"
+      profile = "provenance_run_crate"
+   }
+ }
+}
+```
+
+Note that Research Objects identify people using an ORCID iD, which anyone can
+get for free at [the ORCID website](https://orcid.org/).
+
+The package will be created in the same bucket as the `outdir`, with the package
+name inferred from the S3 key. For example, if the key is
+`my/s3/folder/ro-crate-manifest.json`, the package name will be `my_s3/folder`.
+
+## SQS Message Processing
+
+The primary interface to the Packaging Engine is through an SQS queue in the
+same account and region as your Quilt stack, listed as `PackagerQueueArn` and
+`PackagerQueueUrl` under the Outputs tab. The URL will be something
+like:
+
+```text
+https://sqs.us-east-1.amazonaws.com/XXX/stack-name-PackagerQueue-XXX
+```
+
+The body of the message is the stringified JSON of a package description.
+There is only one required parameter:
+
+```json
+{
+  "source_prefix": "s3://data_bucket/source/folder/metadata.json"
+}
+```
+
+This is assumed to be a folder if it ends in a `/`; otherwise, we will remove
+the last component of the path to get the folder. The contents of the folder
+will be used to create a package in the same bucket as the source folder, with the
+package name being inferred from the source URI.
+
+Optionally, you can control the package name, metadata, and other settings by
+explicitly specifying any of the following fields:
+
+```jsonc
+{
+  "source_prefix": "s3://data_bucket/source/folder/", // trailing '/' for folder
+  "registry": "package_bucket", // may be the same as `data_bucket`
+  "package_name": "prefix/suffix",
+  "metadata": { "key": "value" }, // object (or metadata URI, but not both)
+  "metadata_uri": "metadata.json", // S3 URI to read, relative or absolute
+  "commit_message": "Commit message for the package revision", // string
+  "workflow": "alpha", // name of a valid metadata workflow
+}
+```
+
+The job will fail if you try to specify both `metadata` and `metadata_uri`.
+
+### SendMessage API
+
+If you have appropriate IAM permissions, and the SQS URL, you can send a message
+to the queue using the AWS SDK or the AWS CLI. Here is an example using the AWS
+CLI:
+
+<!--pytest.mark.skip-->
+```bash
+export QUEUE_URL=https://sqs.us-east-1.amazonaws.com/XXX/PackagerQueue-XXX
+aws sqs send-message --queue-url $QUEUE_URL \
+--message-body '{"source_prefix":"s3://data_bucket/source/folder/"}'
+```
+
+## Custom EventBridge Rules
+
+EventBridge rules can be used to transform EventBridge events from any bus in
+your account into a conforming SQS message.
+
+### Example: Event-Driven Packaging (EDP)
+
+[Event-Driven Packaging](../advanced-features/event-driven-packaging.md) is a
+high-end add-on to Quilt that coalesces multiple S3 uploads into a single
```

Comment:
what is a high-end add-on?

Reply:
Switching to "private preview", as it is called in the EDP docs.